### ISSUE TITLE
Duplicate notifications when subscribed to both an organization and repository

### ISSUE THREAD

**Issue by tedivm** [integrations/slack#801]
When subscribing to an organization for the first time it might make sense to clear all of the subscriptions already active in that channel for that organization, as the system will otherwise send duplicate messages.

**Comment by say8425** [comment]
Did you check `/github subscribe list`?
I had similar issue, but I found I subscribed a organization and organization repositories at the same time.

**Comment by tedivm** [comment]
Yup, that's what happened (I even mentioned this in the title)- my thought was that if someone was already subscribed to some repos in an org, and then subscribed to the whole org, it would make sense to clear out those duplicate ones.

**Comment by softmarshmallow** [comment]
in my case, 
- org/repo1
- org/repo2
and
- org

is subscribed.

but only org/repo2 has duplicated message while repo1 doesnt.

**Comment by offero** [comment]
> Did you check `/github subscribe list`?
> I had similar issue, but I found I subscribed a organization and organization repositories at the same time.

I had to come to this issue to know I could list out the subscriptions. /github help did not show that info.

**Comment by offero** [comment]
Just ran into the same issue. I'm not sure how we had both on. Maybe a warning would help. /shrug

**Comment by santoshD93** [comment]
I am facing the same issue.

**Comment by santoshD93** [comment]
I am using Heroku which is sending all notifications to Slack (HerokuChatOps) so I won't be needing this app/integration anymore.
Hope this helps someone.

**Comment by antonkro** [comment]
Same here `/github subscribe list ` shows following

```
user/repo
org/repo2
```

but from repo2 we are getting duplicate messages in the channel

**Comment by Jacobsolawetz** [comment]
Bump on clearing their already subscribed subscriptions within the organization they are trying to subscribe to

Our slack is blowing up!

**Comment by dalehenrich** [comment]
I found that I was receiving duplicate commit messages for an organize. Unsubscribed and resubscribed and I'm still receiving duplicate commit messages ... pretty annoying that you guys don't care about this issue (apparently) and pretty annoying ... I just happen to be one of the lucky few  who will be ignored ... nice work :(

**Comment by hellwolf** [comment]
me too, darn it

**Comment by pkmital** [comment]
I have this issue when moving a github repo to another organization. The slack channel receives messages twice with both listing the new organization's repo. subscribe list shows only the one repo. If I unsubscribe and re-subscribe, I get two messages still. And I cannot remove the repo from the permissions as it does not exist. Any ideas?

**Comment by wturon** [comment]
@pkmital I experienced the same issue. I transferred a repo from one org to another and started receiving duplicate notifications. Completely removing the Github Slack app, reinstalling it, and then resubscribing to the new repo resolved the issue for me.
